app/HutchManager/Controllers/UsersController.cs

```diff
@@ -25,8 +25,8 @@ public async Task<List<UserModel>> List()
   [HttpPost]
   public async Task<IActionResult> Create([FromBody] UserModel user)
   {
-    await _user.Create(user);
-    return Ok(user);
+    
+    return Ok(await _user.Create(user));
```

Comment:
⛏️ Should we do something to handle when the user isn't created successfully? Does this just send back error 500 if that happens? Or does `_user.Create` generate an appropriate code like 4xx if the user somehow sends bad data?

Reply:
Just checked this and it already sends back the appropriate code

Reply:
for info, the service does not return http status codes; that's not it's job. the controller is responsible for that. That said, the web framework does an awful lot of sensible default heavy lifting.

in ASP.NET Core, it is already being handled as a result of strongly typed model binding. The process is approximately as follows:

1. Request comes in with a route and a verb
2. aspnet middleware either terminates early or it hits MVC which looks for a matching controller action
3. (typically aspnet returns 404 if it finds nothing) actually for us it returns the SPA for any unmatched routes, and the SPA instead 404s.
4. it tries to bind the request body to the controller action's method arguments
5. if it fails it automatically returns 400.
6. THEN it starts actually executing the code in the controller action
7. Obviously you're free here to handle any special cases that should return specific status codes, but in this case there aren't really any?
8. Any unhandled exceptions return 500